Ports/glib/patches/fix-gio-meson.build.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/gio/meson.build b/gio/meson.build
+index 849ea8057..49a37a7bd 100644
+--- a/gio/meson.build
++++ b/gio/meson.build
+@@ -13,7 +13,7 @@ gnetworking_h_conf = configuration_data()
+ 
+ gnetworking_h_nameser_compat_include = ''
+ 
+-if host_system not in ['windows', 'android']
++if host_system not in ['windows', 'android', 'serenity']
+   # Don't check for C_IN on Android since it does not define it in public
+   # headers, we define it ourselves wherever necessary
+   if not cc.compiles('''#include <sys/types.h>
+@@ -34,7 +34,7 @@ endif
+ 
+ network_libs = [ ]
+ network_args = [ ]
++if host_system not in ['windows', 'serenity']
+-if host_system != 'windows'
```

Comment:
`+` before `-`? did `git`/`diff` do that?

```suggestion
-if host_system != 'windows'
+if host_system not in ['windows', 'serenity']
```

Reply:
A suggestion for making these patches easier:
- make a separate clean copy of the thing you're trying to port into dir A
- initialise a git repo in A and just `git add -A`
- change stuff in A, commit into there, `git format-patch <hash of the first commmit>` and move all the generated patches to your `Ports/port/patches` dir

with this flow, you can avoid messing with patches and just deal with your own changes in the nicer UI that git has :)

Reply:
That's a great suggestion, thanks!